data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -72,37 +66,35 @@ public DataController(final DataService dataService,
      * @param dataRequest the request to read the data from a leafResource
      * @return a stream of bytes representing the contents of the resource
      */
-    @PostMapping(value = "/read/chunked", consumes = "application/json", produces = "application/octet-stream")
-    public ResponseEntity<StreamingResponseBody> readChunked(@RequestBody final DataRequest dataRequest) {
+    @PostMapping(value = "/read/chunked", consumes = "application/json", produces = "application/json")
```

Comment:
This definitely doesn't produce json.
```suggestion
    @PostMapping(value = "/read/chunked", consumes = "application/json", produces = "application/octet-stream")
```